AzureIndexer/Stratis.Bitcoin.Features.AzureIndexer/AzureIndexerLoop.cs

```diff
@@ -250,44 +276,62 @@ private BlockFetcher GetBlockFetcher(IndexerCheckpoints indexerCheckpoints, Canc
                     int fromHeight = this.StoreTip.Height + 1;
                     int toHeight = Math.Min(this.StoreTip.Height + IndexBatchSize, this.indexerSettings.To);
 
+                    this.logger.LogTrace("Starting indexing from {0}, to {1}", fromHeight, toHeight);
+
                     // Index a batch of blocks
                     if (!cancellationToken.IsCancellationRequested && toHeight > this.BlocksFetcher._LastProcessed.Height)
                     {
+                        this.logger.LogTrace("Indexing blocks");
+
                         this.BlocksFetcher.FromHeight = Math.Max(this.BlocksFetcher._LastProcessed.Height + 1, fromHeight);
                         this.BlocksFetcher.ToHeight = toHeight;
-                        var task = new IndexBlocksTask(this.IndexerConfig);
+                        var task = new IndexBlocksTask(this.IndexerConfig, this.loggerFactory);
                         task.SaveProgression = !this.indexerSettings.IgnoreCheckpoints;
                         task.Index(this.BlocksFetcher, this.AzureIndexer.TaskScheduler);
+
+                        this.logger.LogTrace("Finished blocks");
                     }
 
                     // Index a batch of transactions
                     if (!cancellationToken.IsCancellationRequested && toHeight > this.TransactionsFetcher._LastProcessed.Height)
                     {
+                        this.logger.LogTrace("Indexing txes");
```

Comment:
Are 'txes' a standard way of saying transactions? Or could you say 'txns' or just 'transactions'?

Reply:
>Are 'txes' a standard way of saying transactions?

Not sure. TX is a transaction for sure and that's a standard way but I don't know if for multiple standard is txes or txns. So I used what first popped in my mind 🙂